yabi/yabi/settings.py

```diff
@@ -33,14 +33,13 @@
 # set debug, see: https://docs.djangoproject.com/en/dev/ref/settings/#debug
 DEBUG = env.get("debug", not PRODUCTION)
 
-# django-secure
+# https://docs.djangoproject.com/en/1.8/ref/middleware/#django.middleware.security.SecurityMiddleware
 SECURE_SSL_REDIRECT = env.get("secure_ssl_redirect", PRODUCTION)
-# We do clickjacking support using Django's middleware. See MIDDLEWARE_CLASSES
-SECURE_FRAME_DENY = False
+SECURE_SSL_HOST = env.get("secure_ssl_host", False)
```

Comment:
This should be a hostname not a boolean, so the False default doesn't look right.